app/controllers/authentication_controller.rb

```diff
@@ -0,0 +1,27 @@
+class AuthenticationController < ApplicationController
+  before_action :load_user, only: :login
+
+  # POST /auth/login
```

Comment:
no comment in code

Reply:
Vậy không cần comment chi luôn hả a?
Như vài function nhìn khó hiểu?